datasets/emit-landfill.data.mdx

```diff
@@ -0,0 +1,59 @@
+---
+id: landfill-emit
+name: "EMIT Landfill"
+description: "FILL IN"
+media:
+  src: ::file ./smog-city.png
+  alt: Smog Located In City.
+  author:
+    name: Nick van den Berg
+    url: https://unsplash.com/photos/2vb-_3t6YCM
+taxonomy:
+  - name: Topics
+    values:
+      - Environmental Justice
+layers:
+  - id: landfill-emit
+    stacCol: landfill-emit
+    name: CH4 (ppmm)
+    type: raster
+    description: "Methane (CH4) plumes - EMIT"
+    initialDatetime: newest
+    zoomExtent:
+      - 0
+      - 20
+    sourceParams:
+      colormap_name: plasma_r
+      rescale:
+        - 0
+        - 255
+      nodata: 0
+    compare:
+      datasetId: svi-overall
+      layerId: social-vulnerability-index-overall
+      mapLabel: |
+        ::js ({dateFns, datetime, compareDatetime}) => {
+          return `${dateFns.format(datetime, 'LLL yyyy')} VS ${dateFns.format(compareDatetime, 'LLL yyyy')}`;
+        }
+    legend:
+      type: gradient
+      min: "0"
+      max: "2418"
+      stops:
+        - '#08041d'
+        - '#1f0a46'
+        - '#52076c'
+        - '#f57c16'
+        - '#f7cf39'
+
+
+  
+---
+
+<Block>
+  <Prose>
+
+FILL IN

```

Comment:
Oh, how did I miss this before? This should be filled in, too. https://deploy-preview-379--visex.netlify.app/data-catalog/landfill-emit